Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -32,6 +32,24 @@ class MultilineFormatter: PrettyFormatter {
         """
     }
 
+    func tupleString(elements: [(String?, String)]) -> String {
+        // if the labels of tuples are not specificated, they are like ".1" (not nil).
+        // Using "." as the first charactor of the label of tuple is prohibited.
+        let labelValuePairs: [String] = elements.map { label, value in
+            if let label = label, label.first != "." {
```

Comment:
Missed in b4676ce maybe :)

```suggestion
        let labelValuePairs: [String] = elements.map { label, value in
            if let label = label {
```

Reply:
Thank you!